src/ontology/uberon-edit.obo

```diff
@@ -182428,6 +182428,7 @@ intersection_of: part_of UBERON:0001044 ! saliva-secreting gland
 [Term]
 id: UBERON:0035073
 name: duct of eccrine sweat gland
+def: "The main function of the duct of the eccrine sweet gland is the reabsorption of Na and Cl ions as sweat flows through the duct. It has two cell layers: basal and luminal. It is continuous with the secretory coil and goes towards the skin surface. The distal segment of the duct is relatively straight and connects with the acrosyringium in the epidermis. Most of the ion reabsorption occurs in the proximal part of the duct." [PMID:31608304]
```

Comment:
- While this seems correct, [it is noted in the issue](https://github.com/obophenotype/uberon/issues/2915#issue-1747169486) that this edit was intended for UBERON:0000423.

- Consider adding this first sentence: "A duct made up of simple tubular epithelium that is part of an eccrine sweat gland."

-  "... duct of the eccrine sweet gland... " --> "...  duct of the eccrine **sweat** gland... "